editor/src/components/canvas/controls/insertion-plus-button.tsx

```diff
@@ -275,6 +278,9 @@ const PlusButton = betterReactMemo('PlusButton', (props: ButtonControlProps) =>
     },
     [dispatch, parentPath, indexPosition],
   )
+
+  const ButtonSize = 12
```

Comment:
Was this supposed to be scaled too?

Reply:
nah, I'm using zoom and transorm: scale in this one because of the text